src/app/core/components/directives/details-height/details-height.directive.ts

```diff
@@ -48,6 +48,7 @@ export class IxDetailsHeightDirective implements OnInit, OnDestroy, OnChanges {
 
     this.element.nativeElement.style.height = this.heightCssValue;
     this.window.addEventListener('scroll', this.onScrollHandler, true);
+    setTimeout(() => this.onScroll());
```

Comment:
Why is `setTimeout` needed?

Reply:
There is an issue with height detection if you switches pages between apps and dataset.